libfreefare/mifare_key_deriver.c

```diff
@@ -0,0 +1,276 @@
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+
+#include <openssl/des.h>
+
+#include <freefare.h>
+#include "freefare_internal.h"
+
+#define AN10922_DIV_AES128      0x01
+#define AN10922_DIV_AES192_1    0x11
+#define AN10922_DIV_AES192_2    0x12
+#define AN10922_DIV_2K3DES_1    0x21
+#define AN10922_DIV_2K3DES_2    0x22
+#define AN10922_DIV_3K3DES_1      0x31
+#define AN10922_DIV_3K3DES_2      0x32
+#define AN10922_DIV_3K3DES_3      0x33
```

Comment:
Feel free to align values here :stuck_out_tongue_winking_eye:

Reply:
ok